autonomy/data/Dockerfiles/tendermint/tendermint.py

```diff
@@ -98,6 +98,45 @@ def __str__(self) -> str:
             ")"
         )
 
+    def build_node_command(self, debug: bool = False) -> List[str]:
+        """Build the 'node' command."""
+        p2p_seeds = ",".join(self.p2p_seeds) if self.p2p_seeds else ""
+        cmd = [
+            "tendermint",
+            "node",
+            f"--proxy_app={self.proxy_app}",
+            f"--rpc.laddr={self.rpc_laddr}",
+            f"--p2p.laddr={self.p2p_laddr}",
+            f"--p2p.seeds={p2p_seeds}",
+            f"--consensus.create_empty_blocks={str(self.consensus_create_empty_blocks).lower()}",
+        ]
+        if debug:
+            cmd.append("--log_level=debug")
+
+        if self.home is not None:  # pragma: nocover
+            cmd += ["--home", self.home]
+        return cmd
+
+    @staticmethod
+    def get_node_command_kwargs(monitoring: bool = False) -> Dict:
+        """Get the node command kwargs"""
```

Comment:
Moved these two methods here as proposed by @Karrenbelt to facilitate the mocking.